apps/fabric-website-resources/test/FocusZone.test.dom.ts

```diff
@@ -0,0 +1,42 @@
+import * as puppeteer from 'puppeteer';
+
+describe('FocusZone', () => {
+  let browser: any;
+  let page: any;
+
+  beforeAll(async () => {
```

Comment:
Is there any way this file can live with the FocusZone?

Reply:
BTW this is awesome! I just want to make sure the tests somehow live with close to the source. If it's not possible, what about a separate package for functional tests? (like the ssr-tests package.)

Reply:
There are 2 things I find as issues for functional tests living close to the source:

1. we would then have to add a dev dep on a pretty thing for oufr itself - maybe not a big deal in a monorepo, but strange on principle.

2. this test is highly dependent on the site itself - while we do have the examples living next to the component, it is a little weird to have OUFR lib itself point to a URL that is at Demo site which is an implicit dependency of the demo from OUFR.

I like the idea of having a functional test app package that maybe is divorced from the demo example app. However, I think @apneer is just putting this test out there to collect feedback for the workflow a bit. I think the intention is to move this to a separate app under `/apps` in the future.

One more mental model thing about functional / integration tests: they may actually need to test out interactions of several components with each other. That wouldn't neatly fall under a single component directory because they cross over several.